src/games/strategy/triplea/ui/mapdata/MapData.java

```diff
@@ -193,43 +188,65 @@ public MapData(final ResourceLoader loader) {
     }
   }
 
+  @Override
   public void close() {
     m_resourceLoader.close();
   }
 
-  private void loadTerritoryNames() {
-    m_territoryNameImages = new HashMap<>();
+  private Map<String, Image> territoryNameImages() {
     if (!m_resourceLoader.hasPath("territoryNames/")) {
-      return;
+      return Collections.EMPTY_MAP;
     }
+
+    Map<String, Image> territoryNameImages = new HashMap<>();
     for (final String name : m_centers.keySet()) {
-      try {
-        final Image img = loadImage("territoryNames/" + name + ".png");
-        if (img != null) {
-          m_territoryNameImages.put(name, img);
+        Optional<Image> territoryNameImage = loadTerritoryNameImage(name);
+
+        if (territoryNameImage.isPresent()) {
+          territoryNameImages.put(name, territoryNameImage.get());
         }
-      } catch (final Exception e) {
-        // skip that territory then
-      }
     }
+    return territoryNameImages;
   }
 
-  private void loadDecorations() throws IOException {
-    final URL decorations = m_resourceLoader.getResource(DECORATIONS_FILE);
-    if (decorations == null) {
-      m_decorations = Collections.emptyMap();
-      return;
+  private Optional<Image> loadTerritoryNameImage(String imageName) {
+    Image img = null;
+    try {
+      // try first file names that have underscores instead of spaces
+      final String normalizedName = imageName.replace(' ', '_');
+      img = loadImage(constructTerritoryNameImagePath(normalizedName));
+      if (img == null) {
+        img = loadImage(constructTerritoryNameImagePath(imageName));
+      }
```

Comment:
The normalized one is the common case.

Below is a good approximate count of how many *png files we have with underscores and spaces:
```
dan@dan-desk:~/work/maps$ grep " " < <(find . -name "*png" -type f) | wc -l
1251
dan@dan-desk:~/work/maps$ grep "_" < <(find . -name "*png" -type f) | wc -l
99916
```
(note, counts are for 134 maps, that is most every map, but some very new ones might  not be included)